lib/compiler-llvm/src/translator/intrinsics.rs

```diff
@@ -70,11 +71,44 @@ pub struct Intrinsics<'ctx> {
     pub ctpop_i32: FunctionValue<'ctx>,
     pub ctpop_i64: FunctionValue<'ctx>,
 
+    pub fp_rounding_md: MetadataValue<'ctx>,
+    pub fp_exception_md: MetadataValue<'ctx>,
+
+    pub add_f32: FunctionValue<'ctx>,
+    pub add_f64: FunctionValue<'ctx>,
+    pub add_f32x4: FunctionValue<'ctx>,
+    pub add_f64x2: FunctionValue<'ctx>,
+
+    pub sub_f32: FunctionValue<'ctx>,
+    pub sub_f64: FunctionValue<'ctx>,
+    pub sub_f32x4: FunctionValue<'ctx>,
+    pub sub_f64x2: FunctionValue<'ctx>,
+
+    pub mul_f32: FunctionValue<'ctx>,
+    pub mul_f64: FunctionValue<'ctx>,
+    pub mul_f32x4: FunctionValue<'ctx>,
+    pub mul_f64x2: FunctionValue<'ctx>,
+
+    pub div_f32: FunctionValue<'ctx>,
+    pub div_f64: FunctionValue<'ctx>,
+    pub div_f32x4: FunctionValue<'ctx>,
+    pub div_f64x2: FunctionValue<'ctx>,
+
     pub sqrt_f32: FunctionValue<'ctx>,
     pub sqrt_f64: FunctionValue<'ctx>,
     pub sqrt_f32x4: FunctionValue<'ctx>,
     pub sqrt_f64x2: FunctionValue<'ctx>,
 
+    pub max_f32: FunctionValue<'ctx>,
+    pub max_f64: FunctionValue<'ctx>,
+    pub max_f32x4: FunctionValue<'ctx>,
+    pub max_f64x2: FunctionValue<'ctx>,
+
+    pub min_f32: FunctionValue<'ctx>,
+    pub min_f64: FunctionValue<'ctx>,
+    pub min_f32x4: FunctionValue<'ctx>,
+    pub min_f64x2: FunctionValue<'ctx>,
+
```

Comment:
> All function calls done in a function that uses constrained floating point intrinsics must have the `strictfp` attribute.
> All function definitions that use constrained floating point intrinsics must have the `strictfp` attribute.
-- https://llvm.org/docs/LangRef.html#constrained-floating-point-intrinsics

Please add the `strictfp` attribute and place it in all places where it's required, which includes the trampolines.